wee_slack.py

```diff
@@ -2938,8 +2938,7 @@ def format_nick(nick):
 
 def tag(tagset, user=None):
     if user:
-        user.replace(" ", "_")
-        default_tag = "nick_" + user
+        default_tag = "nick_" + user.replace(" ", "_")
```

Comment:
Hm, I can't see any place the user would contain spaces that needs to be replaced here. Did you experience any problems before this fix?

Reply:
Yes. The RSS bot posts messages with spaces in the nick.